src/main/java/net/snowflake/client/jdbc/SnowflakeSQLLoggedException.java

```diff
@@ -69,100 +67,119 @@ private void sendOutOfBandTelemetryMessage(JSONObject value, SnowflakeSQLLoggedE
    * @param value ObjectNode containing information specific to the exception constructor that
    *     should be included in the telemetry log, such as SQLState or reason for the error
    * @param ex The exception being thrown
+   * @param ibInstance Telemetry instance through which telemetry log will be sent
    * @return true if in-band telemetry log sent successfully or false if it did not
    */
-  private Future<Boolean> sendInBandTelemetryMessage(
-      ObjectNode value, SnowflakeSQLLoggedException ex) {
+  private static Future<Boolean> sendInBandTelemetryMessage(
```

Comment:
I don't see a way for the customer to opt out of this. Can you point me where that logic is? If it's not in we absolutely need to add that. I think in Python we're approaching it like a connection parameter.  Also it should be turned off by default.

Reply:
@sfc-gh-mknister can you point me to the part where we have added flags to disable telemetry?